Changes

```diff
@@ -31,6 +31,9 @@ Working version
 
 ### Tools:
 
+- #9127, #9130: ocamldoc: fix the formatting of closing brace in record types.
+  (David Allsopp, report by @sanette)
```

Comment:
Sorry for the nitpick, but are you confident that this is the preferred "full name" for the contributor? If yes, please record this intent in the `.mailmap` file, whose very last section records pseudonymity choices; if no, someone should ask the contributor.

Reply:
No, I'm not at all sure - I meant to ping @sanette to ask, so thanks for nit!

Reply:
It doesn't really matter to me. That's ok to put my full name (you can find it for 
 instance here: https://github.com/sanette/bogue/blob/master/LICENSE)

Reply:
If you don't care, I would follow the standard practice of using a full name rather than just a username. It is more consistent with the rest of the changelog; and someday you may find it convenient to be able to easily trace back contributions to the project, for example to show a potential employer.